src/content/data/ua_overrides.jsm

```diff
@@ -25,6 +25,24 @@ const UAOverrides = [
       let prefix = originalUA.substr(0, originalUA.indexOf(")") + 1);
       return `${prefix} AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.98 Safari/537.36`;
     }
+  },
+
+  /*
+   * Bug 1480710 - m.imgur.com - Build UA override
+   * WebCompat issue #13154 - https://webcompat.com/issues/13154
+   *
+   * imgur returns a 404 for requests to CSS and JS file if requested with a Fennec
+   * User Agent. By removing the Fennec identifies and adding Chrome Mobile's, we
+   * receive the correct CSS and JS files.
+   */
+  {
+    baseDomain: "imgur.com",
+    applications: ["fennec"],
+    uriMatcher: (uri) => uri.includes("m.imgur.com"),
+    uaTransformer: (originalUA) => {
+      let prefix = originalUA.substr(0, originalUA.indexOf(")") + 1);
+      return prefix + " AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.85 Mobile Safari/537.36";
```

Comment:
Is it worth adding an FxQuantum/version rider to the final string? If so you can probably grab the version number directly from the originalUA as I'm doing here: https://github.com/wisniewskit/google-search-fixer/blob/master/common.js

Reply:
Unfortunately, this seems to result in a blank page again, as the assets 404 with an added string. :(